### ISSUE TITLE
Feature request: Ability to add (not replace) updated time

### ISSUE THREAD

**Issue by riabiy** [beaussan/update-time-on-edit-obsidian#66]
In my head it's quite useful to have a list of updates/times you've touched the note. 

I'd be happy to avoid overriding and just to attach "udpated" time, e.g.

```yaml
updated:
  - 2022-12-12
  - 2023-12-12
  - ....
```

Ofc not everybody needs it, but being able to turn this on in advanced settings would be quite useful. 

Yes, it may get bloated, but I can easily hide field, if needed/remove update. 

But not required, ofc.

**Comment by plasmabit** [comment]
I like this idea. I also thought about it. It is interesting to save info to see how frequently a note was edited.
You might like the compromise that I came up with then: using Git. I got it set up this way that it only tracks *.md, and .gitignore files. Then using `restore` command I can make the files appear as they were on a chosen commit. That allows me to see if a note was edited on a commit, which gives me a rough idea how frequently it was edited in total. You can also schedule Git commits to be every day or every week.

**Comment by riabiy** [comment]
> I like this idea. I also thought about it. It is interesting to save info to see how frequently a note was edited.
> 
> You might like the compromise that I came up with then: using Git. I got it set up this way that it only tracks *.md, and .gitignore files. Then using `restore` command I can make the files appear as they were on a chosen commit. That allows me to see if a note was edited on a commit, which gives me a rough idea how frequently it was edited in total. You can also schedule Git commits to be every day or every week.

Yep. Another option - using restoration tool. For non-tec-savy people. Yet, having this in my front-matter is where it makes the most sense to me. I want it to be somewhat in my face. Moreover, it should not be a hard change. Sooo...🤷‍♂️

**Comment by gmmazza** [comment]
I second this append option. There is another plugin that does this with a YAML checkbox to selected notes from [alangrainer](https://github.com/alangrainger/obsidian-frontmatter-modified-date). It could be great to unify everything in one plugin.

**Comment by riabiy** [comment]
> I second this append option. There is another plugin that does this with a YAML checkbox to selected notes from [alangrainer](https://github.com/alangrainger/obsidian-frontmatter-modified-date). It could be great to unify everything in one plugin.

Does it behave like I've described? I mean multiple updates as a list.
